lib/Models/Cesium3DTilesCatalogItem.ts

```diff
@@ -30,27 +30,11 @@ import createStratumInstance from "./createStratumInstance";
 import Feature from "./Feature";
 import Mappable from "./Mappable";
 import proxyCatalogItemUrl from "./proxyCatalogItemUrl";
-
-class ObservableCesium3DTileset extends Cesium3DTileset {
-  _catalogItem?: Cesium3DTilesCatalogItem;
-  @observable destroyed = false;
-
-  destroy() {
-    super.destroy();
-    // TODO: we are running later to prevent this
-    // modification from happening in some computed up the call chain.
-    // Figure out why that is happening and fix it.
-    runLater(() => {
-      runInAction(() => {
-        this.destroyed = true;
-      });
-    });
-  }
-}
+import Cesium3dTilesMixin from "../ModelMixins/Cesium3dTilesMixin";
 
 export default class Cesium3DTilesCatalogItem
   extends FeatureInfoMixin(
-    AsyncMappableMixin(
+    Cesium3dTilesMixin(
```

Comment:
The top of the file still references `Cesium3DCatalogItemTraits.ts` which I presume is no longer required?

Reply:
In fact there are a squillion imports which I guess aren't need anymore

Reply:
Done